src/features/client/components/RestaurantListCard/index.jsx

```diff
@@ -0,0 +1,92 @@
+import {
+  Box,
+  Flex,
+  Text,
+  Heading,
+  Image,
+  LinkBox,
+  LinkOverlay,
+  Center,
+} from '@chakra-ui/react';
+import { Link } from 'react-router-dom';
+
+import bicycleUrl from '../../../../assets/bicycle-icon.svg';
+
+function RestaurantListCard({ restaurant }) {
+  return (
+    <LinkBox
+      as={Link}
+      width="100%"
+      maxW={{ base: '100%', md: '49%', lg: '32%' }}
+      borderWidth="1px"
+      rounded="md"
+      overflow="hidden"
+      display="flex"
+      flexDir="column"
+      gap="2"
+      to="/restaurantinspect"
```

Comment:
in real world, route should be called `/restaurants/:id` to follow REST conventions. If it's possible, please refactor. If not keep in mind the convention for later BE integration